topics/training/tutorials/create-new-tutorial-content/tutorial.md

```diff
@@ -108,21 +115,21 @@ Once we set up the infrastructure, we are ready to write the tutorial
 
 ## Finding a good toy dataset
 
-The first question to come is which data using for the tutorial and particularly for the hands-on parts. This data must be informative enough to illustrate the use of a tool or a technic, but it must not be too big to be able to be run during workshop or locally. Typically, this is a subset of a full dataset where the informative data has been extracted.
+The first question to come is what data to use for waltking the tutorial through the hands-on sections. The selected data must be informative enough to illustrate the meaning of using a tool or a given technique, but not too big to reqire long waiting times for its processing during a workshop. Typically, the selected data should be the informative subset of a full real-life dataset.
```

Comment:
"waltking"?

Reply:
ah, right, thanks. It should be "walking"